packages/playwright-test/src/cli.ts

```diff
@@ -44,6 +44,7 @@ export function addTestCommand(program: Command) {
   command.option('--global-timeout <timeout>', `Maximum time this test suite can run in milliseconds (default: unlimited)`);
   command.option('-j, --workers <workers>', `Number of concurrent workers, use 1 to run in a single worker (default: number of CPU cores / 2)`);
   command.option('--list', `Collect all the tests and report them, but do not run`);
+  command.option('--list-files', `High level list`);
```

Comment:
Why both this argument, and a separate command?

Reply:
leftover, thanks.